scripts/west_commands/packages.py

```diff
@@ -0,0 +1,145 @@
+# Copyright (c) 2024 Basalte bv
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import os
+import subprocess
+import sys
+import textwrap
+from itertools import chain
+from pathlib import Path
+
+from west.commands import WestCommand
+from zephyr_ext_common import ZEPHYR_BASE
+
+sys.path.append(os.fspath(Path(__file__).parent.parent))
+import zephyr_module
+
+
+def in_venv() -> bool:
+    return sys.prefix != sys.base_prefix
+
+
+class Packages(WestCommand):
+    def __init__(self):
+        super().__init__(
+            "packages",
+            "manage packages for Zephyr",
+            "List and Install packages for Zephyr and projects",
+            accepts_unknown_args=True,
+        )
+
+    def do_add_parser(self, parser_adder):
+        parser = parser_adder.add_parser(
+            self.name,
+            help=self.help,
+            description=self.description,
+            formatter_class=argparse.RawDescriptionHelpFormatter,
+            epilog=textwrap.dedent(
+                """
+            Listing packages:
+
+                Run 'west packages <manager>' to list packages for Zephyr and active projects
+                for a given package manager.
+                See 'west packages <manager> --help' for details.
+            """
+            ),
+        )
+
+        parser.add_argument(
+            "-p",
+            "--project",
+            action="append",
+            default=[],
+            dest="projects",
+            help="Project to run the 'packages' command for. "
```

Comment:
```suggestion
            help="west project to run the 'packages' command for. "
```
"project" is incredibly vague (and yes, I know that ship has sailed)

Reply:
I think `project` is fine, as this is a west extension command and `west` itself just uses the term `Project`.
```
$ west help update
...
positional arguments:
  PROJECT               projects (by name or path) to operate on; defaults to all active projects
```

Note, `west project` is also fine by me, I just wanted to inform that `west` itself just uses the term `project`.

Reply:
Updated everything to "module"